docs/tutorials/basic/symbol.md

```diff
@@ -45,6 +45,19 @@ For a visual explanation of these concepts, see
 To make things concrete, let's take a hands-on look at the Symbol API.
 There are a few different ways to compose a `Symbol`.
 
+## Prerequisites
+
+To complete this tutorial, we need:
+
+- MXNet. See the instructions for your operating system in [Setup and Installation](http://mxnet.io/get_started/install.html)
+- [Python](https://www.python.org/downloads/)
+- [Graphviz](http://www.graphviz.org/)
```

Comment:
This will go on install page as a part of https://github.com/dmlc/mxnet/pull/6617.